src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -706,6 +706,19 @@ void regclass_graph_Model(py::module m) {
         return "<" + class_name + ": '" + self.get_friendly_name() + "'\ninputs[\n" + inputs_str + "\n]\noutputs[\n" +
                outputs_str + "\n]>";
     });
+
+    model.def(
+        "__deepcopy__",
```

Comment:
Can this class have a whole copy-interface? `__copy__` can be raised with error - does it make sense?